libs/designsystem/tsconfig.spec.json

```diff
@@ -1,63 +1,64 @@
-      "@kirbydesign/designsystem/flag": ["flag/index.ts"],
-      "@kirbydesign/designsystem/form-field": ["form-field/index.ts"],
-      "@kirbydesign/designsystem/spinner": ["spinner/index.ts"],
-      "@kirbydesign/designsystem/toggle": ["toggle/index.ts"],
-      "@kirbydesign/designsystem/item": ["item/index.ts"],
-      "@kirbydesign/designsystem/section-header": ["section-header/index.ts"],
-      "@kirbydesign/designsystem/slide": ["slide/index.ts"],
-      "@kirbydesign/designsystem/toggle-button": ["toggle-button/index.ts"],
-      "@kirbydesign/designsystem/tabs": ["tabs/index.ts"],
-      "@kirbydesign/designsystem/tab-navigation": ["tab-navigation/index.ts"],
-      "@kirbydesign/designsystem/slide-button": ["slide-button/index.ts"],
-      "@kirbydesign/designsystem/range": ["range/index.ts"],
-      "@kirbydesign/designsystem/popover": ["popover/index.ts"],
-      "@kirbydesign/designsystem/loading-overlay": ["loading-overlay/index.ts"],
-      "@kirbydesign/designsystem/divider": ["divider/index.ts"],
-      "@kirbydesign/designsystem/checkbox": ["checkbox/index.ts"],
-      "@kirbydesign/designsystem/accordion": ["accordion/index.ts"],
-      "@kirbydesign/designsystem/radio": ["radio/index.ts"],
-      "@kirbydesign/designsystem/item-sliding": ["item-sliding/index.ts"],
-      "@kirbydesign/designsystem/list": ["list/index.ts"],
-      "@kirbydesign/designsystem/avatar": ["avatar/index.ts"],
-      "@kirbydesign/designsystem/progress-circle": ["progress-circle/index.ts"],
-      "@kirbydesign/designsystem/router-outlet": ["router-outlet/index.ts"],
-      "@kirbydesign/designsystem/item-group": ["item-group/index.ts"],
-      "@kirbydesign/designsystem/calendar": ["calendar/index.ts"],
-      "@kirbydesign/designsystem/button": ["button/index.ts"],
-      "@kirbydesign/designsystem/empty-state": ["empty-state/index.ts"],
-      "@kirbydesign/designsystem/dropdown": ["dropdown/index.ts"],
-      "@kirbydesign/designsystem/page": ["page/index.ts"],
-      "@kirbydesign/designsystem/header": ["header/index.ts"],
-      "@kirbydesign/designsystem/modal": ["modal/index.ts"],
-      "@kirbydesign/designsystem/modal/v2": ["modal/v2/index.ts"],
-      "@kirbydesign/designsystem/kirby-app": ["kirby-app/index.ts"],
-      "@kirbydesign/designsystem/chart": ["chart/index.ts"],
-      "@kirbydesign/designsystem/helpers/scss": ["helpers/scss/index.ts"],
-      "@kirbydesign/designsystem/fab-sheet": ["fab-sheet/index.ts"],
-      "@kirbydesign/designsystem/data-table": ["data-table/index.ts"],
-      "@kirbydesign/designsystem/reorder-list": ["reorder-list/index.ts"],
-      "@kirbydesign/designsystem/toast": ["toast/index.ts"],
-      "@kirbydesign/designsystem/action-group": ["action-group/index.ts"],
-      "@kirbydesign/designsystem/menu": ["menu/index.ts"],
-      "@kirbydesign/designsystem/badge": ["badge/index.ts"]
+      "@kirbydesign/core": ["libs/core/src/index.ts"],
```

Comment:
We are only allowed to define path aliases for code that is located within the `rootDir`.
So now that rootDir is the workspace root to include the libs/core package, we need to redefine all the paths.